lambda_uploader/uploader.py

```diff
@@ -171,3 +174,16 @@ def _validate_package_size(self, pkg):
         '''
         if path.getsize(pkg) > MAX_PACKAGE_SIZE:
             LOG.warning("Size of your deployment package is larger than 50MB!")
+
+    def _format_vpc_config(self):
+        '''
+        Returns {} if the VPC config is set to None by Config,
+        returns the formatted config otherwise
+        '''
+        if self._config.raw['vpc']:
```

Comment:
So here you should be able to pull vpc from self._config without calling raw. I assume by this point that the config is loaded since it gets loaded on initialization.